synchronization/src/androidTest/java/de/cyface/synchronization/TestUtils.java

```diff
@@ -25,14 +25,14 @@
     /**
      * An username used by the tests to set up a Cyface account for synchronization.
      */
-    final static String DEFAULT_USERNAME = "test";
+    final static String DEFAULT_USERNAME = "test@cyface.de";
     /**
      * A password used by the tests to set up a Cyface account for synchronization.
      */
-    final static String DEFAULT_PASSWORD = "secret";
+    final static String DEFAULT_PASSWORD = "7vXnwVAXfWXKbC32";
```

Comment:
Please do not provide passwords on s2 on github. An please change that one now.

Reply:
See above: It's a special account for integration testing only. Although it's a dirty workaround. Depending on weather I should disable the API dependent tests I can just remove this account then and delete the credentials here.